ports/libcgroup/vcpkg.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "libcgroup",
+  "version-semver": "3.1.0",
+  "description": "Library for working with cgroup",
+  "homepage": "https://github.com/libcgroup/libcgroup",
+  "license": "LGPL-2.1",
```

```suggestion
  "license": "LGPL-2.1-only",
```
Do not just apply my suggestion. Also run `vcpkg format-manifest ports/libcgroup/vcpkg.json`. 
(There is a GH action for this, and it would flag certain issues. But it didn't run yet.)